### ISSUE TITLE
Re-export FORM_ERROR / ARRAY_ERROR

### ISSUE THREAD

**Issue by vladshcherbin** [final-form/react-final-form#262]
### Are you submitting a **bug report** or a **feature request**?

A feature request

### What is the current behavior?

```js
import { Form, Field } from 'react-final-form'
import { FORM_ERROR } from 'final-form'
```

### What is the expected behavior?

```js
import { FORM_ERROR, Form, Field } from 'react-final-form'
```

### Other information

Maybe this was intentional, I don't know.

I just stumbled this a few times and didn't understand, why the whole form error is not set until debugged `FORM_ERROR` as `undefined` when imported from `react-final-form`

**Comment by erikras** [comment]
Hmm.. I never considered that. `FORM_ERROR` must live in the `final-form` package, but I guess `react-final-form` could export it as well... Presumably it would nudge the bundle size a little higher, but I wonder if the trade-off for reduced dev confusion is worth it. 🤔

**Comment by vladshcherbin** [comment]
@erikras yeah, I had this in two different projects: one long time ago, second today. Totally forgot about it and had to scratch my head again. So I wondered if I'm not alone with this one :)

Maybe you can test it locally to see if it adds size to the bundle. I'm okay with both your decisions on this one, it's just a small thing cool to have. ✋
